_journal-configs/cell.yaml

```
@@ -0,0 +1,26 @@
+name: Cell
+publisher: Cell Press / Elsevier
+url: https://www.cell.com/cell/authors
+
+sizes:
+  single_column: { width_mm: 85,  max_height_mm: 225 }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)